node_modules/gh-users/lib/rest.js

```diff
@@ -107,6 +107,7 @@ GrassHopper.appRouter.on('get', '/api/me', function(req, res) {
     if (req.ctx.user) {
         me = _.extend(me, {'anon': false}, req.ctx.user.toJSON());
     }
+    me = _.extend(me, {'termsAndConditions': UsersAPI.getTermsAndConditionsStatus(req.ctx)});
```

Comment:
I don't think we need to assign this to `me` as it will update the first argument.